front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart.js

```diff
@@ -18,7 +18,7 @@ import {
 import { useDispatch, useSelector } from 'react-redux';
 
 import { CgLoadbar } from 'react-icons/cg';
-import ChartModal from 'applications/operationalStudies/components/SimulationResults/ChartModal';
+import ChartModal from 'applications/operationalStudies/components/SimulationResults/ChartModal/ChartModal';
```

Comment:
You can have a file named `index.js` to import directly `applications/operationalStudies/components/SimulationResults/ChartModal`

Reply:
Done